discojs/discojs-node/src/models/model_loader.ts

```diff
@@ -0,0 +1,23 @@
+import fs from 'node:fs'
+import fsPromises from 'node:fs/promises'
+import { serialization, models } from '@epfml/discojs-core'
+
+export async function saveModelToDisk(model: models.Model, modelFolder: string, modelFileName: string): Promise<void> {
+  try {
+    if (!fs.existsSync(modelFolder)) {
+      fs.mkdirSync(modelFolder)
+    }
+  } catch (err) {
+    console.error(err);
+  }
```

Comment:
let it fail (it would anyway when writing the file but it's more precise here)

```suggestion
  if (!fs.existsSync(modelFolder)) {
    fs.mkdirSync(modelFolder)
  }
```